src/styles/What.css

```diff
@@ -320,7 +320,7 @@
     }
 }
 
-@media (max-width: 1800px) and (min-width: 751px){
+@media (max-width: 900px) and (min-width: 751px){
```

Comment:
What.css: The `@media (max-width: 900px) and (min-width: 751px)` is reflected to allow devices with such a resolution to be displayed on the screens.